docs: add animations documentation for the beta

fbasso opened the pull request.

maxokorokov commented on demo/src/app/pages/animations/animations.component.ts:
```diff
@@ -0,0 +1,32 @@
+
+@Component({templateUrl: './animations.component.html'})
+export class AnimationsPage {
+  rightExample = Snippet(
```
I think you forgot to remove this one

jnizet commented on demo/src/app/pages/animations/animations.component.html:
```diff
@@ -0,0 +1,79 @@
+            Beta testing starting with <span
+                class="ml-1 badge badge-info align-middle font-weight-normal">7.0.0-beta.1</span>
+        </h4>
+        Animations will coming with ng-bootstrap 7. They are already available in the beta 7, so feel free to use it and
```
will come? will be coming?

jnizet commented on demo/src/app/pages/animations/animations.component.html:
```diff
@@ -0,0 +1,79 @@
+    <ngbd-code lang="typescript" [snippet]="config"></ngbd-code>
+
+    <p>
+        You can also choose to disabled them by widget type, by changing the widget configuration:
```
disabled -> disable

jnizet commented on demo/src/app/pages/animations/animations.component.html:
```diff
@@ -0,0 +1,79 @@
+    </ul>
+    <ngbd-page-header title="Found an issue ?" fragment="issues"></ngbd-page-header>
+    <p>
+        Playing with animations has been an hard work, as we had to manage a lots of internal things asynchronously when
```
an hard -> a hard
a lots -> a lot

fbasso commented:
Thanks for the review @jnizet ! I will fix the typos.

benouat commented:
@fbasso as we are shifting animations to v8, would you mind updating quickly the couple references we have in this doc page, so we could merge it for v7 ?

fbasso commented:
@benouat, sure, I'll have a look.

fbasso commented:
I changed the version to 8, and rebased on master to get the latest changes.

Regarding the information, I'm not sure about this:

> More generally, changing the widget state with a model binding will not trigger an animation. You will need to run the widget api instead. For example for the nav, use select to change the nav programmatically.

We will need to check this point (to see if it's applicable to every widget) after their review and integration in the animation branch.

maxokorokov commented:
Fixed some typos, added `npm install @ng-bootstrap/ng-bootstrap@next` snippet, removed some phrases that we've actually fixed :) Will merge when green.
